example-config.yaml

```diff
@@ -77,6 +93,17 @@ stormpath:
             placeholder: "Last Name"
             required: true
             type: "text"
+          # Unless specifically set to false, the organizationNameKey field will
+          # automatically be shown when stormpath.web.multiTenancy.enabled=true,
+          # and an organization cannot be resolved (the default organization
+          # resolver does not return an organization).
+          organizationNameKey:
+            enabled: null
```

Comment:
I'm ok with having this as part of the fields for the reasons robert pointed out.